unit/analyses/variable-sensitivity/eval-member-access.cpp

```diff
@@ -0,0 +1,111 @@
+#include <testing-utils/use_catch.h>
+
+#include <analyses/variable-sensitivity/abstract_enviroment.h>
+#include <analyses/variable-sensitivity/abstract_object.h>
+#include <analyses/variable-sensitivity/constant_array_abstract_object.h>
+#include <analyses/variable-sensitivity/variable_sensitivity_object_factory.h>
+#include <util/arith_tools.h>
+#include <util/mathematical_types.h>
+
+exprt make_array(
+  std::vector<int> contents,
+  abstract_environmentt &environment,
+  namespacet &ns
+);
+
+exprt fetch_element(
+  int index,
+  exprt &array,
+  abstract_environmentt &environment,
+  namespacet &ns
+);
+
+SCENARIO(
```

Comment:
Unit tests are good.  In theory much of VSD should be testable via unit tests so having more is a good thing.